### PR TITLE
Added Server2Server configuration as requirement #2971

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added a proper configuration on server-to-server authentication is required.
#2971

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance

@joaolloureiro Please check the comment and let us know if that good enough or if you would like to add something.

**Comment by joaolloureiro** [comment]
Looks good to me. Thank for quick reply on this.

Thanks
Joao

File: Skype/SfbHybrid/hybrid/plan-call-data-connector.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ The following requirements assume that you already have Skype for Business Serve
 > [!IMPORTANT]
 > Call Data Connector will not function if Monitoring is not enabled on the front end pool.
 
+- A required Server-to-Server authentication configuration properly [see more](https://docs.microsoft.com/skypeforbusiness/manage/authentication/server-to-server-and-partner-applications).
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/plan-call-data-connector.md]:
```suggestion
- Properly configured [server-to-server authentication](https://docs.microsoft.com/skypeforbusiness/manage/authentication/server-to-server-and-partner-applications). 
```

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@CarolynRowe Following up on this PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**